tests/chgres_cube/ftst_program_setup.F90

```diff
@@ -19,30 +19,27 @@ program ftst_program_setup
   if (my_rank .eq. 0) print*, "Starting test of program_setup."
   if (my_rank .eq. 0) print*, "testing read_setup_namelist with file fort.41..."
   call read_setup_namelist()
-  if (cycle_mon .ne. 7 .or. cycle_day .ne. 4 .or. cycle_hour .ne. 12) stop 4
-  if (.not. convert_atm .or. .not. convert_sfc .or. .not. convert_nst) stop 5
-  if (regional .ne. 0 .or. halo_bndy .ne. 0 .or. halo_blend .ne. 0) stop 6
-  if (trim(mosaic_file_target_grid) .ne. "/scratch1/NCEPDEV/da/George.Gayno/noscrub/reg_tests/chgres_cube/fix/C96/C96_mosaic.nc") stop 7
-  if (trim(fix_dir_target_grid) .ne. "/scratch1/NCEPDEV/da/George.Gayno/noscrub/reg_tests/chgres_cube/fix/C96/fix_sfc") stop 8
-  if (trim(orog_dir_target_grid) .ne. "/scratch1/NCEPDEV/da/George.Gayno/noscrub/reg_tests/chgres_cube/fix/C96/") stop 9
-  if (trim(vcoord_file_target_grid) .ne. "/scratch1/NCEPDEV/da/George.Gayno/ufs_utils.git/UFS_UTILS/reg_tests/chgres_cube/../../fix/fix_am/global_hyblev.l64.txt") stop 10
-  if (trim(data_dir_input_grid) .ne. "/scratch1/NCEPDEV/da/George.Gayno/noscrub/reg_tests/chgres_cube/input_data/fv3.nemsio") stop 11
-  if (trim(atm_files_input_grid(1)) .ne. 'gfs.t12z.atmf000.nemsio') stop 12
-  if (trim(sfc_files_input_grid(1)) .ne. 'gfs.t12z.sfcf000.nemsio') stop 13
-  if (varmap_file .ne. "NULL") stop 14
-  if (thomp_mp_climo_file .ne. "NULL") stop 16
-  if (trim(cres_target_grid) .ne. "C96") stop 17
-  if (atm_weight_file .ne. "NULL") stop 18
-  if (trim(input_type) .ne. "gaussian_nemsio") stop 19
-  if (trim(external_model) .ne. "GFS") stop 20
-  if (num_tracers .ne. 7) stop 21
-  if (tracers(1) .ne. "sphum" .or. tracers(2) .ne. "liq_wat" .or. tracers(3) .ne. "o3mr" .or. &
-       tracers(4) .ne. "ice_wat" .or. tracers(5) .ne. "rainwat" .or. tracers(6) .ne. "snowwat" .or. &
-       tracers(7) .ne. "graupel") stop 22
+  if (cycle_mon .ne. 10 .or. cycle_day .ne. 3 .or. cycle_hour .ne. 0) stop 3
+  if (.not. convert_atm .or. .not. convert_sfc .or. .not. convert_nst) stop 4
+  if (regional .ne. 0 .or. halo_bndy .ne. 0 .or. halo_blend .ne. 0) stop 5
+  if (trim(mosaic_file_target_grid) .ne. "/scratch1/NCEPDEV/da/George.Gayno/noscrub/reg_tests/chgres_cube/fix/C192/C192_mosaic.nc") stop 6
+  if (trim(fix_dir_target_grid) .ne. "/scratch1/NCEPDEV/da/George.Gayno/noscrub/reg_tests/chgres_cube/fix/C192/fix_sfc") stop 7
+  if (trim(orog_dir_target_grid) .ne. "/scratch1/NCEPDEV/da/George.Gayno/noscrub/reg_tests/chgres_cube/fix/C192/") stop 8
+  if (trim(vcoord_file_target_grid) .ne. "/scratch1/NCEPDEV/da/George.Gayno/ufs_utils.git/UFS_UTILS/reg_tests/chgres_cube/../../fix/fix_am/global_hyblev.l64.txt") stop 9
```

Comment:
I see you had hardcoded personal account paths before and still have them here now. So this code only works for testing on Hera?

Reply:
This is just a unit test that checks the read of the program namelist. It uses a dummy namelist. The paths are not used.